apps/discovery_api/priv/static/tableau/connector.js

```diff
@@ -24,36 +26,52 @@ window.DiscoveryWDCTranslator = {
   convertDatasetRowToTableRow: _convertDatasetRowToTableRow
 };
 
-function _setupConnector(tableauInstance) {
-  var connector = tableauInstance.makeConnector();
+function submit(mode) {
+  connectionData = {mode}
```

Comment:
maybe a var to make it clear this is a declaration

Reply:
done